ciri/BUILD

```diff
@@ -40,6 +41,11 @@ cc_binary(
     ],
 )
 
+cc_image(
```

Comment:
The `cc_image` contains only ciri and runtime dependencies, not a container, you cannot configure ciri via `cc_image`. 
the optional flags do not work in README.md, in this case, you should set up `container_image` as well.
> bazel run -c opt --define network=mainnet -- ciri:ciri_image # optional flags

Reply:
After a few testing, `cc_image` should have [distroless](https://console.cloud.google.com/gcr/images/distroless/GLOBAL/cc) image. But it does lack some more info. I will set up `container_image` and give more comprehensive instruction in `README`.